app/retail/templates/leaderboard.html

```diff
@@ -14,46 +15,98 @@
     <div class="row no-gutter  text-uppercase">
         <div class="col-3" style="background: #eee; ">
             {% include 'shared/current_balance.html' %}
-            {% include 'shared/sidebar_search.html' %}
+            <!-- {% include 'shared/sidebar_search.html' %} -->
         </div>
-
         <div class="col body">
             <div class="container-fluid" id="leaderboard">
             {% include 'shared/bounty_def.html' %}
-                <div class="row mt-0 ml-1 pt-1 pl-1">
-                    <select id="key">
```

Comment:
i think well likely want to hide the 'key' field and just standardize on the monthly leaderboard for now.   anyone disagree?